pywal/util.py

```diff
@@ -123,12 +123,16 @@ def rgb_to_hex(color):
 
 def darken_color(color, amount):
     """Darken a hex color."""
+    if not isinstance(amount, float):
```

Comment:
I can update the usage of these throughout the program to take ints if it means removing these lines from these function.

Reply:
That would be awesome, thanks. I'll remove it now. For the sake of backwards compatability, I haven't been able to make it accept float arguments yet. Everytime it hits a dot, in the string format, it looks for an attribute named like the text after the dot, so I end up with an error like: `colors has no attribute '5)'`. Ideally I'd like to make it accept floats too. Do you have any idea how to do that? Perferably avoiding having to do something like `{background.darken_color(0\.5)}`.

Also, I was thinking that maybe the color editing functions should be loaded from the scripts directory, that way, users can implement their own color modifying functions. Could open up some neat configs, like an organised rainbow theme down the line.